scan/lib/scan/runner.rb

```diff
@@ -52,6 +52,11 @@ def test_app
       end
 
       command = @test_command_generator.generate
+
+      execute(command: command, devices: Scan.devices)
```

Comment:
Rerunning this throws an error with 👇 

```sh

xcodebuild: error: Existing file at -resultBundlePath "/Users/josh/Projects/fastlane/test-ios-2/fastlane/test_output/FastlaneApp.xcresult" 
```

I think I need to look at a way to remove this existing `xcresult` for the retries 🤷‍♂️

Reply:
Got a fix for this 💪